packages/merge-conflicts/src/browser/style/index.css

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
```

Comment:
this file could be removed?

Reply:
right, it's a leftover from decorators try outs.